features/built_in_matchers/satisfy.feature

```diff
@@ -8,9 +8,14 @@ Feature: `satisfy` matcher
     expect(7).not_to satisfy { |v| v % 5 == 0 }
     ```
 
-  This flexibility comes at a cost, however: the failure message ("expected [actual] to satisfy
-  block") is not very descriptive or helpful.  You will usually be better served by using one of
-  the other built-in matchers, or writing a custom matcher.
+  The default failure message ("expected [actual] to satisfy block") is not very descriptive or helpful.
+  To add clarification, you can provide your own description as an argument:
+
+    ```ruby
+    expect(11).to satisfy("have a remainder of 0 with modulo 5") do |v|
```

Comment:
This seems like a verbose way to say "be evenly divisible by 5" or "be a multiple of 5".  Thoughts on using one of those shorter descriptions?